cloudbuild-prod.yaml

```diff
@@ -27,6 +27,15 @@ steps:
     args: [ 'run', 'deploy', 'sds', '--image', 'europe-west2-docker.pkg.dev/ons-sds-dns/sds/sds-prod:latest',
             '--region', 'europe-west2', '--allow-unauthenticated', '--ingress', 'internal-and-cloud-load-balancing' ]
 
+  - name: 'gcr.io/cloud-builders/gcloud'
+    id: 'Show image vulnerabilities'
+    entrypoint: sh
+    args:
+      - '-c'
+      - |
+        gcloud alpha container images describe europe-west2-docker.pkg.dev/${_PROJECT_ID}/sds/sds-prod:$SHORT_SHA \
```

Comment:
The image file for prod should be europe-west2-docker.pkg.dev/ons-sds-dns/sds/sds-prod:latest
Similar question as above

Reply:
Sorry this isn't ready for review yet. I am still working on a few bits

Reply:
Updated the image tag to latest however, in this step both $SHORT_SHA & latest points to the same image in the run so won't make any difference with either tag.